docs/versioned_docs/version-v3.0.0/02-getting-started.md

```diff
@@ -0,0 +1,95 @@
+* *message pump:* an Arcus Messaging-provided registered service that receives Azure Service Bus messages for you, and "pumps" them through your *message handlers*.
+:::
+
+## The basics
+:::note
+Arcus Messaging is currently only supported for [Azure Service Bus](https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-messaging-overview) solutions.
+:::
+
+Arcus Messaging helps with receiving messages from a message broker. This walkthrough uses Azure Service Bus as an example.
+
+Instead of directly interacting with the [`ServiceBusReceiver`](https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusreceiver), it allows you to implement one or more *'message handler'* interfaces. Arcus Messaging will use these custom implementations and determine based on several criteria to which  *'message handler'* it should route the message.
+
+By using *'message handlers'*, you don't have to worry about routing, deserialization, or even complete/dead-letter/abandon messages - but you can take care of them as well.
+
+![Message handling schema](/media/worker-message-handling.png)
+
+## Implement your first message handler
+First step in creating your message handler, is installing the following package. This is the only package that is required during this walkthrough.
+
+```powershell
+PS> Install-Package -Name Arcus.Messaging.ServiceBus
+```
+
+The package makes the `IServiceBusMessageHandler<>` interface available. Implementing this interface is the simplest way of creating a *'message handler'*.
+
+As the generic type, you can use the DTO (data-transfer object) to which the [`ServiceBusReceivedMessage.Body`](https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusreceivedmessage.body) should be deserialized to (default via JSON). In this case: `MyOrder`.
+
+```csharp
+using Arcus.Messaging.ServiceBus;
+
+public class MyOrder
+{
+    public string OrderId { get; set; }
+    public string ProductName { get; set; }
+}
+
+// highlight-start
+public class MyOrderMessageHandler : IServiceBusMessageHandler<MyOrder>
+// highlight-end
+{
+    private readonly ILogger _logger;
+
+    public MyOrderMessageHandler(ILogger<MyOrderMessageHandler> logger)
+    {
+        _logger = logger;
+    }
+
+// highlight-start
+    public async Task ProcessMessageAsync(
+        MyOrder order,
+        ServiceBusMessageContext context,
+        MessageCorrelationInfo correlation,
+        CancellationToken cancellation)
+// highlight-end
+    {
+        // Process further your custom type...
+    }
+}
+```
+
+## Register your message handlers
+A messaging solution usually has more than one type of message it receives. Differentiating between types, properties, can be a hassle to set up.
+
+*'Message handlers'* are registered on a *'message pump'* in your application. This 'pump' receives the Azure Service Bus messages for you and routes them to the right handler.
```

Comment:
Actually, the pump 'retrieves' the message from the message broker and routes it to the correct message handler.
I'd use the verb 'retrieve' instead of 'receive'